tests/acceptance/default_ncf.cf.sub

```diff
@@ -9,9 +9,26 @@ bundle common ncf_inputs
      # long string to be built, causing CFEngine to error out, so we rebuild
      # the full paths using maplist below
      "generic_ncf_files"
-        string => execresult("/bin/bash -c \"cd ${ncf_tree} && /usr/bin/find 10_ncf_internals/configuration.cf 10_ncf_internals/expand_configuration.cf 20_cfe_basics 30_generic_methods 40_it_ops_knowledge -name '*.cf' -printf '%p\n'\"", "noshell");
+        string => execresult("/bin/bash -c \"cd ${ncf_tree} && /usr/bin/find 10_ncf_internals/configuration.cf 10_ncf_internals/expand_configuration.cf 20_cfe_basics 30_generic_methods 40_it_ops_knowledge -name '*.cf' -print\"", "noshell");
```

Comment:
```find```'s ```-printf``` option is not available on AIX 5.3. However, it is not needed, ```-print``` is just fine, and what we already use in ```list-compatible-inputs```.